docs/ref/models/fields.txt

```diff
@@ -1215,6 +1215,47 @@ when :attr:`~django.forms.Field.localize` is ``False`` or
     information on the difference between the two, see Python's documentation
     for the :mod:`decimal` module.
 
+``GeneratedField``
+------------------
+
+.. versionadded:: 5.0
+
+.. class:: GeneratedField(expression, db_persist=None, output_field=None, **kwargs)
+
+A field which is computed based on other fields in the model. Similar to a
+database view, this field is managed and updated by the database itself. Uses
+the ``GENERATED`` SQL syntax.
+
```

Comment:
```suggestion

.. admonition:: Refresh the data
    Since the database calculates the value and the `save` method does not return anything, after saving the model it will be necessary to retrieve the model from the database again (e.g.: `refresh_from_db`) to access the new value of the generated field.

```

Reply:
Added.